docs/src/examples/components/Menu/Types/MenuExampleWithSubmenu.shorthand.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { Menu } from '@stardust-ui/react'
+
+const menuWidthVariable = {
+  menuItemWidth: 100,
+}
+
+const fileSubmenu = {
+  items: [
+    { key: 'new', content: 'New' },
+    { key: 'open', content: 'Open' },
+    { key: 'edit', content: 'Edit' },
+  ],
+}
+const editSubmenu = {
+  items: [
+    { key: 'undo', content: 'Undo' },
+    { key: 'redo', content: 'Redo' },
+    { key: 'cut', content: 'Cut' },
+    { key: 'copy', content: 'Copy' },
+  ],
+}
+const formatSubmenu = {
+  items: [{ key: 'font', content: 'Font' }, { key: 'text', content: 'Text' }],
+}
+
+const items = [
+  { key: 'file', content: <span>File ▾</span>, submenu: fileSubmenu },
+  { key: 'edit', content: <span>Edit ▾</span>, submenu: editSubmenu },
+  { key: 'format', content: <span>Format ▾</span>, submenu: formatSubmenu },
+  { key: 'help', content: 'Help' },
+]
+
+const MenuExampleWithSubmenu = () => (
+  <Menu defaultActiveIndex={0} items={items} variables={menuWidthVariable} />
+)
```

Comment:
Please try to add some height on the element as the content is not visible (try to wrap the Menu in some wrapper with some bigger height). 
![image](https://user-images.githubusercontent.com/4512430/45699812-2e734080-bb6b-11e8-925a-9ff864f37de1.png)

Reply:
Also, I would expect to be able to expand only one submenu at a time. I don't think the scenario above (having three different dropdowns should work (try closing the submenu if there is click outside of it).